**********************************************************************
File "./dbmanager.py", line 28, in __main__.DbManager
Failed example:
    print(db)
Expected:
    TABLE: table1
    id           coulumn1     coulumn2   
    ------------ ------------ ------------
    2            Miyagi       Sendai     
    3            Tokyo        Shinjuku   
Got:
    TABLE: table1
    id           coulumn1     coulumn2    
    ------------ ------------ ------------
    2            Miyagi       Sendai      
    3            Tokyo        Shinjuku    
**********************************************************************
1 items had failures:
   1 of  13 in __main__.DbManager
***Test Failed*** 1 failures.
